### ISSUE TITLE
can you convert VGM to MIDI on Linux ?

### ISSUE THREAD

**Issue by journeytosilius** [JamesDunne/vgm2midi#1]
Can I use this to convert VGM to MIDI on linux ? just asking because I can't see it on the documentation, since it's related to the game emulator higan but not for this tool. Thanks

**Comment by JamesDunne** [comment]
Yes. Start by compiling from the higan directory with make. It'll produce something in the out folder called vgm2midi.

Kind of have to read the code to figure out what it wants for input files but it works quite well in my experience with a properly configured input file. I'll try to dig up some of my old example files and upload and document when I have time.

I believe this version works for both nes and snes. best results with nes usually. snes tries to analyze the instrument samples to determine the base frequency anf adjust midi notes from there but it can get it wrong by an octave when an overtone is stronger than the fundamental.

**Comment by journeytosilius** [comment]
thanks ! I will try it asap
